src/components/MapView/Analyser/AnalysisTable/index.tsx

```diff
@@ -112,6 +121,12 @@ function AnalysisTable({ classes, tableData, columns }: AnalysisTableProps) {
         page={page}
         onChangePage={handleChangePage}
         onChangeRowsPerPage={handleChangeRowsPerPage}
+        labelRowsPerPage={safeTranslate(t, 'Rows Per Page')}
+        labelDisplayedRows={({ from, to, count }) => {
+          return `${from}–${to} ${safeTranslate(t, 'of')} ${
```

Comment:
a comment would be nice here, looks complex. Can't tell why we are checking -1 just by looking

Reply:
this is temporary, I added a small comment